reference/docfx.json

```diff
@@ -49,7 +49,7 @@
         "overwrite": [],
         "externalReference": [],
         "globalMetadata": {
-            "breadcrumb_path": "bread/toc.yml",
+            "breadcrumb_path": "/powershell/scripting/bread/toc.json",
```

Comment:
The file is YAML not json. That is not a valid file path. How is that going to work? I have this working in 5 other docsets in this repo. That is not the pattern of configuration that is working in the other docsets.